playbooks/roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -7,6 +7,12 @@
     remote: upstream
   with_items: "{{ hammer_devel_repositories }}"
 
+- name: 'Add fork remotes to cloned repositories'
```

Comment:
@akofink This works, but it might be cleaner to have a command prior to it that runs `git remote` and then allow `git remote add` to check the output to see if the name of the remote exists already within the output or not.